.github/workflows/nucliadb_telemetry_release.yml

```diff
@@ -25,6 +22,7 @@ jobs:
 
       - name: Install requirements
         run: |
+          python bump.py --build=${{github.run_number}}
```

Comment:
I would keep telemetry on its own package inside de same repo

Reply:
Yes, I thought about that but I guess didn't want to have to deal with some many special cases for management and releasing on this repo.

We have many pipelines and complexity packed into the testing, packaging and releasing.